AdobeStockAsset/etc/db_schema.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0"?>
```

Comment:
let's remove the db_schema from this pull request as it will be delivered with https://github.com/magento/adobe-stock-integration/pull/50

Reply:
ups